app/controllers/contacts_controller.rb

```diff
@@ -0,0 +1,14 @@
+class ContactsController < ApplicationController
+
+  def create
+    @company = Company.find(params[:contact][:company_id])
+    @contact = @company.contacts.create(contact_params)
```

Comment:
same here, what if a user fails to enter a contact's email? can they still create a contact? (i don't think the specs asked for this, but it's something to keep in mind!)